app/Http/Controllers/CampController.php

```diff
@@ -1027,11 +1027,20 @@ public function getCampBreadCrumb(Request $request, Validate $validate)
         $filter['asOf'] = $request->as_of;
         $filter['asOfDate'] = $request->as_of_date;
         $filter['campNum'] = $request->camp_num;
+        $data = new stdClass();
+        $data->flag=0;
+        $data->subscription_id=null;
+        $data->subscribed_camp_name=null;
         try {
             $livecamp = Camp::getLiveCamp($filter);
-            $data = new stdClass();
             $data->bread_crumb = Camp::campNameWithAncestors($livecamp, $filter);
-            $indexs = ['bread_crumb'];
+            if ($request->user()) {
+                $campSubscriptionData = Camp::getCampSubscription($filter, $request->user()->id);
+                $data->flag = $campSubscriptionData['flag'];
+                $data->subscription_id = isset($campSubscriptionData['camp_subscription_data'][0]['subscription_id']) ? $campSubscriptionData['camp_subscription_data'][0]['subscription_id'] : null;
+                $data->subscribed_camp_name = isset($campSubscriptionData['camp_subscription_data'][0]['camp_name']) ? $campSubscriptionData['camp_subscription_data'][0]['camp_name'] : null;
```

Comment:
`$data->subscribed_camp_name = $campSubscriptionData['camp_subscription_data'][0]['camp_name'] ??  : null;`

How about this?